Dashboard/Views/StatusView.axaml

```diff
@@ -1,8 +1,246 @@
-             x:Class="Dashboard.Views.StatusView">
-    Status Page Test
-</UserControl>
\ No newline at end of file
+             x:Class="Dashboard.Views.StatusView"
+             x:DataType="vm:StatusViewModel"
+             Foreground="Black"
+             Background="#c4c4c4">
+
+    <UserControl.Styles>
+        <Style Selector="TabItem">
+            <Setter Property="FontSize" Value="18"/>
+            <Setter Property="Height" Value="34"/>
+            <Setter Property="VerticalAlignment" Value="Center"/>
+            <Setter Property="Background" Value="#a3a3a3"/>
+            <Setter Property="Margin" Value="0 0 0 0"/>
+            <Setter Property="Padding" Value="10 0"/>
+            <Setter Property="FontWeight" Value="Bold"/>
+        </Style>
+        <Style Selector="TabItem:pointerover">
+            <Setter Property="Foreground" Value="White"/>
+        </Style>
+        <Style Selector="TabItem:pointerover /template/ ContentPresenter#PART_ContentPresenter">
+            <Setter Property="VerticalAlignment" Value="Center"/>
+            <Setter Property="Margin" Value="0 0 0 0"/>
+        </Style>
+        <Style Selector="TabItem:selected">
+            <Setter Property="Foreground" Value="Black"/>
+            <Setter Property="Background" Value="#787373"/>
+        </Style>
+        <Style Selector="TabItem:selected /template/ ContentPresenter#PART_ContentPresenter">
+            <Setter Property="Background" Value="#787373"/>
+            <Setter Property="Margin" Value="0 0 0 0"/>
+            <Setter Property="Padding" Value="10 0"/>
+        </Style>
+    </UserControl.Styles>
+
+    <Grid>
+        <Grid.RowDefinitions>
+            <RowDefinition Height="Auto" />
+            <RowDefinition Height="Auto" />
+            <RowDefinition Height="Auto" />
+        </Grid.RowDefinitions>
+        <Grid.ColumnDefinitions>
+            <ColumnDefinition Width="*" />
+            <ColumnDefinition Width="Auto" />
+            <ColumnDefinition Width="*" />
+        </Grid.ColumnDefinitions>
+
+        <Border Grid.Row="1" Grid.Column="1" Background="#d9d9d9" Height="870" Margin="0,20,0,0">
+            <!-- TabControl for AV Data, GPS, and RES -->
+            <TabControl>
+                <!-- AV Data Tab -->
+                <TabItem Header="AV Data">
+                    <Grid>
+                        <Grid.RowDefinitions>
+                            <RowDefinition Height="Auto" />
+                            <RowDefinition Height="Auto" />
+                            <RowDefinition Height="Auto" />
+                        </Grid.RowDefinitions>
+                        <Grid.ColumnDefinitions>
+                            <ColumnDefinition Width="*" />
+                            <ColumnDefinition Width="*" />
+                        </Grid.ColumnDefinitions>
+                        <!-- AV Data Tab Content -->
+                        <StackPanel Orientation="Horizontal" Width="1200">
```

Comment:
I understand that we need to have this width, or some way of ensuring the layout is consistent, but it doesn't fit half a 1080 display (which I would assume could be a reasonable smallest bound)

Reply:
We could consider making an app wide min height / width?

Reply:
I did try to make it say 80% of the screen width but wasn't able to make that work. I am up for suggestions and yeah happy to set app wide constraints for this

Reply:
I think we can revisit this in the future, maybe during final sprint we can touch it up. im not fussed rn